### PR TITLE
Prevents academy wizard from spawning in the wiz den

### PR DESCRIPTION
**Pull request by Guestratem**
## About The Pull Request

Tweaks whether or not the wizard academy defender will be moved to the den or not as per the On gain proc

## Why It's Good For The Game

its just quality of life, for both the admins who don't have to manually move the wizard academy defender to the academy and the players, who will actually spawn there

## Changelog
:cl:
Tweak: prevents moving the academy wizard to the den 
/:cl:

### CODE REVIEW

**Comment by Guestratem** [comment]
I don't know this wasn't done on creation its took me like 2 minutes to put the code in. testing however could've have gone smoother

**Comment by ATH1909** [comment]
Perhaps also consider making a copy of this PR over on /tg/, so that the other downstreams of /tg/ (and /tg/ itself if an admin feels like spawning the wizard academy away mission in) can have their wizard academy away missions work properly.

**Comment by Guestratem** [comment]
its already been done on TG

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -838,7 +838,7 @@
 		if(!src.is_busy && (src.zone_selected == BODY_ZONE_HEAD || src.zone_selected == BODY_ZONE_PRECISE_GROIN) && get_turf(src) == get_turf(T) && !(T.mobility_flags & MOBILITY_STAND) && src.a_intent != INTENT_HELP) //all the stars align, time to curbstomp
 			src.is_busy = TRUE
 
-			if (!do_mob(src,T,25) || get_turf(src) != get_turf(T) || (T.mobility_flags & MOBILITY_STAND) || src.a_intent == INTENT_HELP) //wait 30ds and make sure the stars still align (Body zone check removed after PR #958)
+			if (!do_mob(src,T,25) || get_turf(src) != get_turf(T) || (T.mobility_flags & MOBILITY_STAND) || src.a_intent == INTENT_HELP || src == T) //wait 30ds and make sure the stars still align (Body zone check removed after PR #958)
```
Comment (qwertyquerty) [inline code/modules/mob/living/carbon/human/human.dm]: Curbstomp changes snuck into this PR, fix

**Comment by Guestratem** [comment]
ah, this is a comparison pull request I might need to remove the changes

File: code/modules/mob/living/carbon/human/human.dm (hunk 1 above)
Comment (Guestratem) [reply code/modules/mob/living/carbon/human/human.dm]: fixed, forgot It was a fork comparison PR

**Pull request merged**